scalameta/parsers/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -1660,6 +1663,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
               case Term.Quasi(0, _) => true
               case Term.Quasi(1, ParamLike()) => true
               case NameLike() => true
+              case Term.Ascribe(ParamLike(), _) => true
```

Comment:
This should be as restrictive as possible, i.e. `_: Term.Quasi` instead of `ParamLike()`